tests/benchmark/snappy/dynamic/PerfComparison.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+java -ea -cp "/home/kishor/SNAPPY/CHECKOUT/snappydata/cluster/build-artifacts/scala-2.11/libs/snappydata-cluster_2.11-1.0.0-tests.jar" io.snappydata.benchmark.snappy.TPCHPerfComparer /home/kishor/SNAPPY/OUTPUT/HYDRA/smokePerf/
```

Comment:
Remove hard coded path for checkout from script instead read the value from parameter and replace it at runtime.

Reply:
Same with output path location.  Read the value from parameter and replace it at runtime.